src/fermo/app.py

```diff
@@ -4,8 +4,11 @@
 # internal file imports
 from views import views
 
+
+# initialize the app 
 app = Flask(__name__)
+app.config.from_pyfile('config.py')
```

Comment:
Change to app.config.from_file() and use a toml or json file instead of .py

Reply:
E.g. https://github.com/rochacbruno/flask-project-template/blob/main/settings.toml